cli/cli_test.go

```diff
@@ -125,15 +133,13 @@ func TestJobPlanConflictingDeployment(t *testing.T) {
 	exitCode = runCmd().Run([]string{testPack, testRefFlag})
 	require.Equal(t, 1, exitCode)
 
-	reset()
+	// reset is run in clearJob
```

Comment:
Not blocking, but I'd be ok just deleting these comment lines.

Reply:
I thought the comment might help prevent folks from putting a reset at the end of these tests and re-breaking things.  The other possibility could be moving the reset up to a defer on the cases that don't require `clearJob` so that the end action is always up at the top of the test func.  What do you think?

Reply:
Yeah, I like the idea of the end action in a defer at the top of the function, and then engineers can decide what to call there.